insights_core.te

```diff
@@ -281,6 +281,14 @@ optional_policy(`
 	pcp_write_pid_sock_file(insights_core_t)
 ')
 
+optional_policy(`
+	gen_require(`
+		type rhcd_t;
+	')
+
+	allow insights_core_t rhcd_t:fifo_file { read write };
```

Comment:
We have interfaces for rhcd_t (please do not use "naked" allow rules for types defined in another module -- always use an interface if available).
```
optional_policy(`
    rhcd_read_fifo_files(insights_core_t)
    rhcd_write_fifo_files(insights_core_t)
')
```